packages/amplify-util-uibuilder/src/commands/utils/getApiConfiguration.ts

```diff
@@ -0,0 +1,71 @@
+import { printer } from '@aws-amplify/amplify-prompts';
+import { AmplifyStudioClient } from '../../clients';
+import { $TSContext } from '@aws-amplify/amplify-cli-core';
+import { getUiBuilderComponentsPath } from './getUiBuilderComponentsPath';
+import { getCodegenConfig } from 'amplify-codegen';
+import { ApiConfiguration } from 'aws-sdk/clients/amplifyuibuilder';
+import path from 'path';
+
+export function relativeToComponentsPath(importPath: string, context: $TSContext): string {
+  const componentsPath = getUiBuilderComponentsPath(context);
+  return path.relative(componentsPath, importPath).split(path.sep).join('/');
```

Comment:
Does this work on as expected on Windows?

I think it would be better to use `path.join` to combine the path pieces into a single string.

Reply:
This works as expected and is actually because windows is the "exception" to the norm. These strings get dropped into jsx files as import paths, so the need to be built with `/` instead of `\` like we would get when running on windows

Reply:
That makes sense, thanks for explaining! Can you leave a comment explaining this in the code for future readers?

I'd prefer we use `path.posix.join` instead if it will work for all platforms, but it's not a merge blocker.

Reply:
I like that suggestion, I'll make sure we try switching to `path.posix.join` and document why this is needed on the next update.